packages/web-app-files/src/components/PublicLinksSidebar/EditPublicLink.vue

```diff
@@ -7,7 +7,7 @@
         name="custom-classes-transition"
       >
         <oc-alert v-if="errors" class="oc-files-file-link-error-alert" variation="danger">
-          {{ errors }}
+          <span v-translate>{{ errors.toString() }}</span>
```

Comment:
@paulcod3 do you remember the outcome of our discussion about translating error messages? 🤔 I don't anymore, sadly

Reply:
This doesn't help with translations, because we also need all possible error messages in transifex / in the translations template first.